VARIABLES.md

```diff
@@ -98,3 +99,28 @@ Name | Description | Type | Default | Required |
 | schema_names | List of Apiary schemas that this role can read/write. | list(string) | - | yes |
 | max_role_session_duration_seconds | Number of seconds that the assumed credentials are valid for.| string | `"3600"` | no |
 | allow_cross_region_access | If `true`, will allow this role to write these Apiary schemas in all AWS regions that these schemas exist in (in this account). If `false`, can only write in this region. | bool | `false` | no |
+
+
+### apiary_managed_schemas
+
+A list of maps. Along with S3 bucket name, each map corresponds to S3 storage properties. 
```

Comment:
This exact same line (106) should be added to the description for `apiary_managed_schemas` on line 13, along with a section link to this section (please do it the same was as `apiary_assume_roles`).

Also, this section comes alphabetically after `apiary_assume_roles`, so it should be moved below.